admin-dev/themes/new-theme/js/components/tinymce-editor.js

```diff
@@ -145,6 +160,58 @@ class TinyMCEEditor {
    * @param editor
    */
   setupEditor(editor) {
+    /* // TODO <cnc-modifica> - TinyMCEEditor::setupEditor() - Creazione pulsante */
+    if (editor.advantageLink) {
+      const allCmsRoute = this.router.generate('admin_cms_pages_all_cms');
+      editor.addButton('advantageLink', {
+        type: 'button',
+        text: '',
+        icon: 'link',
+        title: 'Internal link',
+        onclick() {
+          $.ajax({
+            type: 'POST',
+            url: allCmsRoute,
+            success(data) {
+              const pages = [];
+              $.each(data, (index, cms) => {
+                pages.push({text: cms.meta_title, value: cms.id_cms});
+              });
+
+              editor.windowManager.open({
+                title: 'Aggiungi link interno',
```

Comment:
It's better to haveenglish words here

Reply:
@nicosomb,

Yes, yes, of course, the texts need to be in English. This is just a draft (let's say more of a test to understand how to implement everything).
As suggested by @Hlavtox, I'm trying to think more globally, so that I can also include references for products, categories, and manufacturers.